Manual update to bring tags history to images reference docs

erikaheidi opened the pull request:
This PR brings a manual update to Images reference docs, with the new "Tags History" page.

The image specs page was not touched, because it requires information that is not readily available anymore and requires a `crane` call to a specific image digest. While I'm still working on this update for autodocs, I will manually update the other pages to make sure users aren't left without docs for the new images that are coming every day.

The updated overview page now shows information about `latest` and `latest-dev` tags + digests, instead of the previous "tag aliases".

Preview Links:
- [Reference docs entry point](https://deploy-preview-834--ornate-narwhal-088216.netlify.app/chainguard/chainguard-images/reference/)
- [Overview Page (Go Image)](https://deploy-preview-834--ornate-narwhal-088216.netlify.app/chainguard/chainguard-images/reference/go/overview/)
- [Tags History Page (Go Image)](https://deploy-preview-834--ornate-narwhal-088216.netlify.app/chainguard/chainguard-images/reference/go/tags_history/)

erikaheidi committed:
Manual update to bring tags history to images reference docs

ltagliaferri commented on content/chainguard/chainguard-images/reference/apko/overview.md:
```diff
@@ -1,7 +1,7 @@
+title: "Image Overview: Apko"
 type: "article"
-description: "Overview: apko Chainguard Images"
+description: "Overview: Apko Chainguard Image"
```
How did the `a` in `apko` get title cased?

erikaheidi replied:
Ah! I used a function to title case all image names when generating the pages, forgot about apko and melange specific cases. Should I revert all?

About mobile: ugh! yeah that data is hard to format to look nice on mobile because the digests are so long. I will need some help to figure this one out.

erikaheidi replied:
Fixed this!

ltagliaferri reviewed (approved):
Amazing work, Erika! I think we should bring this in and I wanted to raise the point that on mobile the tags history pages are tough to read right now:

![Recording 2023-07-13 at 14 50 47](https://github.com/chainguard-dev/edu/assets/5977693/4609fb1d-9184-42e5-b5df-7dcb81e19df7)

We have a lot more users on desktop than mobile, but ideally these pages would be accessible to both. Maybe we can bring it up in a team meeting about how to format this page for mobile experience?

erikaheidi committed:
Updating pages

erikaheidi committed:
removing title case from descriptions

erikaheidi commented:
Sent two updates:
- removed title case from titles and descriptions
- changed the relative time (x days ago) to static date in tags history page because it doesn't reflect real time elapsed in static pages. this would need to be implemented in javascript for accurate readings.

jamonation commented:
Here's a PR to set the width of the image digests on the tag history pages to 320px: https://github.com/chainguard-dev/edu/pull/839

jamonation committed:
Add max-width to tag history page image digests (#839)

Signed-off-by: Jamon Camisso <jamonation+git@gmail.com>
